libs/zzz/page-optimize/src/ConditionalToggle.tsx

```
@@ -0,0 +1,87 @@
+
+export function ConditionalToggles({
+  condMetas,
+}: {
+  condMetas: CondMeta[] | CondMeta
+}) {
+  if (!Array.isArray(condMetas)) condMetas = [condMetas]
+  return condMetas.map((c) => <ConditionalToggle key={c.key} condMeta={c} />)
+}
+function ConditionalToggle({ condMeta }: { condMeta: CondMeta }) {
+  const { database } = useDatabaseContext()
+  const character = useCharacterContext()!
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)